nuclear/module/NUClearModule.cmake

```
@@ -191,6 +191,16 @@ function(NUCLEAR_MODULE)
   # Put it in an IDE group for the module's directory
   set_target_properties(${module_target_name} PROPERTIES FOLDER ${module_path})
 
+  # ####################################################################################################################
+  # Warnings #
+  # ####################################################################################################################
+
+  # * We need to tell cmake where the function for the compiler warnings is (src/cmake)
+  list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/nuclear/cmake")
+  include(CompilerWarnings)
+  # * Then we set the warnings with it for the module
+  set_project_warnings(${module_target_name})
```

Or `set_module_warnings`? @TrentHouliston any suggestions on naming scheme?
```suggestion
  set_target_warnings(${module_target_name})
```

---

Or maybe like set default warnings?